tests/components/websocket_api/test_auth.py

```diff
@@ -41,6 +65,30 @@
     assert msg['message'] == 'Invalid access token or password'
 
 
+async def test_auth_events_incorrect_pass(hass, no_auth_websocket_client):
+    """Test authenticating."""
+    connected_evt = []
+    hass.helpers.dispatcher.async_dispatcher_connect(
+        SIGNAL_WEBSOCKET_CONNECTED,
+        lambda: connected_evt.append(1))
+    disconnected_evt = []
+    hass.helpers.dispatcher.async_dispatcher_connect(
+        SIGNAL_WEBSOCKET_DISCONNECTED,
+        lambda: disconnected_evt.append(1))
+
+    await test_auth_via_msg_incorrect_pass(no_auth_websocket_client)
+
+    await asyncio.sleep(0.1)
```

Comment:
Can we do it without sleep? For example call `async_block_till_done()` twice?

Reply:
That blocks forever, the websocket task isn't ever done.

Reply:
That's not actually needed for the test to run. Deleted it.